testing/sandbox.py

```diff
@@ -155,3 +156,26 @@ def tailer_sandbox(port, log_path):
     finally:
         os.kill(proc.pid, signal.SIGTERM)
         proc.wait()
+
+
+def install_fake_uwsgi():
+    # Install a fake uwsgi module into sys.modules
+    # with a few of the used methods defined
+    class uwsgi(object):
+        mule_msg_hook = None
+
+        @staticmethod
+        def mule_msg(message, mule=None):
+            pass
+
+        @staticmethod
+        def mule_msg_recv_size():
+            return 65536
+
+    class uwsgidecorators(object):
+        @staticmethod
+        def mule_msg_dispatcher(message):
+            pass
+
+    sys.modules['uwsgi'] = uwsgi
+    sys.modules['uwsgidecorators'] = uwsgidecorators
```

Comment:
this should at the very least be a context manager which undoes this when exiting.  It should also probably use `mock.patch.dict(...)`

(otherwise this is a good source of test pollution)